kuma/core/tests/test_utils.py

```diff
@@ -21,3 +25,26 @@ def test_empty_string(self):
     def test_wrong_type(self):
         eq_(0, smart_int(None))
         eq_(10, smart_int([], 10))
+
+
+@override_settings(USE_SHORT_URLS=True)
+class BitlyTestCase(KumaTestCase):
```

Comment:
The `KumaTestCase` isn't needed here and we can just use Django's standard test class